platform/osx/display_server_osx.mm

```diff
@@ -1118,14 +1118,14 @@ static int translateKey(unsigned int key) {
 		/* 34 */ KEY_UNKNOWN,
 		/* 35 */ KEY_ESCAPE,
 		/* 36 */ KEY_META,
-		/* 37 */ KEY_META,
-		/* 38 */ KEY_SHIFT,
+		/* 37 */ KEY_META, // Command/Meta on Mac apprears to only have one keycode: 55 (0x37), so left/right cannot be distinguished
```

Comment:
Left Meta is `55 (0x37)`, Right Meta is `54 (0x36)`.

Reply:
Source?

Reply:
I can't find any solid sources for this... A lot of mappings I have found just have 0x36 blank. But I will add it as you say anyway, I have found a couple of places that agree with that layout but like I said nothing solid.

Reply:
Right command on my Logitech MX Keys keyboard do produce 0x36 code on Big Sur, but maybe it's not true for some other keyboards. And it's listed in macOS SDK `/System/Library/Frameworks/Carbon.framework/Versions/A/Frameworks/HIToolbox.framework/Versions/A/Headers/Events.h` as `kVK_RightCommand = 0x36`.

Reply:
Ah great, thanks!